### PR TITLE
chore: update nuka-carousel to latest

### PR DESCRIPTION
**Pull request by ChaoLiangSuper**
- Update the carousel to 4.4.3
- Migrate the API

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Check-list:

- [x] I have read the [Contributing](CONTRIBUTING.md) document.
- [x] I've thought about and labelled my PR/commit message appropriately.
- [x] If this PR introduces breaking changes I've described the impact and migration path for existing applications.
- [x] CI is green (coverage, linting, tests).
- [x] I have updated the documentation accordingly.
- [x] I've two LGTMs/Approvals.
- [x] I've fixed or replied to all my code-review comments.
- [x] I've manually tested with a buddy.
- [x] I've squashed my commits into one.

### CODE REVIEW

File: src/components/adslot-ui/Carousel/index.jsx [hunk 1]
```diff
@@ -1,58 +1,27 @@
...
 
 const baseClass = 'carousel-component';
-const navigationDelay = 600;
-const decoratorStyles = {
-  bottom: 0,
-  width: '30px',
-};
-
-export const getPrevDecorator = () => {
-  let previousSlideThrottled;
```
Comment (ChaoLiangSuper) [inline src/components/adslot-ui/Carousel/index.jsx]: Checked source code the button will be rendered after every click. So the `delay` is not working at all. 
Have manually played the animation, working fine.

**Comment by ChaoLiangSuper** [comment]
Not yet, will test before merge and release @vinteo

**Comment by ChaoLiangSuper** [comment]
Tested by @weilai1988

**Comment by lightbringer1991** [comment]
merging...

**Pull request merged**